app/actuators/ReactiveMap.js

```diff
@@ -580,14 +580,14 @@ export default class ReactiveMap extends Component {
 		});
 
 		return (
-			<div className={`rbc rbc-reactivemap col s12 col-xs-12 card thumbnail ${cx}`} style={ReactiveMapHelper.mapPropsStyles(this.props.componentStyle, "component")}>
+			<div className={`rbc rbc-reactivemap col s12 col-xs-12 card thumbnail ${cx}`} style={ReactiveMapHelper.mapPropsStyles(this.props.style, "component")}>
 				{title}
 				{showMapStyles}
 				<GoogleMapLoader
 					containerElement={
 						<div
 							className="rbc-container col s12 col-xs-12"
-							style={ReactiveMapHelper.mapPropsStyles(this.props.componentStyle, "map", this.mapDefaultHeight)}
```

Comment:
@metagrover The `componentStyle` injection was happening twice in the previous code. Please check if this is right.